webpack.config.js

```diff
@@ -34,6 +34,10 @@ const NO_DEV_SERVER = env.NO_DEV_SERVER;
 const FORCE_WEBPACK_DEV_SERVER = env.FORCE_WEBPACK_DEV_SERVER;
 const IS_CI = !!env.CI || !!env.TRAVIS;
 
+const DEV_MODE = !(IS_PRODUCTION || IS_CI);
+const SHOULD_FORK_TS = DEV_MODE && !Boolean(env.NO_TS_FORK);
```

Comment:
nit, but we do this above:

```suggestion
const SHOULD_FORK_TS = DEV_MODE && !env.NO_TS_FORK;
```

Reply:
Burak told me is that as long as the variable is defined, then it is true ... regardless of the value, but I cannot use it !! because of the cases as below:

![image](https://user-images.githubusercontent.com/29228205/75921348-432bbd00-5e61-11ea-9b28-dbdf98cac67e.png)

Reply:
Ahhh, 👍 , we'll probably want to update the old code to behave the same (in the future).